include/intx/intx.hpp

```diff
@@ -1025,6 +1033,17 @@ struct uint
     friend constexpr bool operator>=(const uint& x, const uint& y) noexcept { return !(x < y); }
     friend constexpr bool operator<=(const uint& x, const uint& y) noexcept { return !(y < x); }
 
+    friend constexpr std::strong_ordering operator<=>(const uint& x, const uint& y) noexcept
+    {
+        if (x < y)
```

Comment:
Use the same pattern: equal then less checks.

Reply:
Oh my bad, resolved